engines/advancedDetector.cpp

```diff
@@ -327,34 +328,57 @@ Common::Error AdvancedMetaEngine::createInstance(OSystem *syst, Engine **engine)
 
 void AdvancedMetaEngine::reportUnknown(const Common::FSNode &path, const ADFilePropertiesMap &filesProps, const ADGameIdList &matchedGameIds) const {
 	Common::String report = Common::String::format(
+			"The game in '%s' seems to be an unknown %s engine game "
+			"variant.\n\nPlease report the following data to the ScummVM "
+			"team at %s along with the name of the game you tried to add and "
+			"its version, language, etc.:",
+			path.getPath().c_str(), getName(), "https://bugs.scummvm.org/");
+	
+	Common::String reportTranslated = Common::String::format(
 			_("The game in '%s' seems to be an unknown %s engine game "
 			  "variant.\n\nPlease report the following data to the ScummVM "
 			  "team at %s along with the name of the game you tried to add and "
 			  "its version, language, etc.:"),
-			path.getPath().c_str(), getName(), "https://bugs.scummvm.org/");
-
+			  path.getPath().c_str(), getName(), "https://bugs.scummvm.org/");
+	
+	Common::String bugtrackerAffectedEngine = Common::String::format(getName());
+	
 	if (matchedGameIds.size()) {
 		report += "\n\n";
-		report += _("Matched game IDs:");
+		reportTranslated += "\n\n";
+		report += "Matched game IDs:";
+		reportTranslated += _("Matched game IDs:");
 		report += " ";
+		reportTranslated += " ";
 
 		for (ADGameIdList::const_iterator gameId = matchedGameIds.begin(); gameId != matchedGameIds.end(); ++gameId) {
 			if (gameId != matchedGameIds.begin()) {
 				report += ", ";
+				reportTranslated += ", ";
 			}
 			report += *gameId;
+			reportTranslated += *gameId;
 		}
 	}
 
 	report += "\n\n";
+	reportTranslated += "\n\n";
 
-	report.wordWrap(80);
+	reportTranslated.wordWrap(65);
 
-	for (ADFilePropertiesMap::const_iterator file = filesProps.begin(); file != filesProps.end(); ++file)
+	for (ADFilePropertiesMap::const_iterator file = filesProps.begin(); file != filesProps.end(); ++file) {
 		report += Common::String::format("  {\"%s\", 0, \"%s\", %d},\n", file->_key.c_str(), file->_value.md5.c_str(), file->_value.size);
-
+		reportTranslated += Common::String::format("  {\"%s\", 0, \"%s\", %d},\n", file->_key.c_str(), file->_value.md5.c_str(), file->_value.size);
+	}
 	report += "\n";
-
+	reportTranslated += "\n";
+	
+	// Show the UnknownGameDialog and print the translated unknown game information
+	GUI::UnknownGameDialog dialog(report, reportTranslated, bugtrackerAffectedEngine);
+	dialog.runModal();
```

Comment:
I wonder what happens when invoking the detector from the command line using say `--detect`.

Reply:
Hm, apparently it quits without printing anything. Unless I'm doing something wrong that's not quite ideal.

Reply:
Erm... it segfaults...

Reply:
To be precise, it segfaults when the detector is invoked from the command line. @bgK do you know how to check if the GUI is currently running? I tried to check for `GUI::GuiManager::instance()._launched == true` and make this the condition to invoke GUI::UnknownGameDialog, but that doesn't work...

Reply:
Fixed in b711c1e (code by @criezy)